com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/FinalPass.shader

```diff
@@ -83,7 +84,7 @@ Shader "Hidden/HDRP/FinalPass"
             UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
             float2 positionNDC = input.texcoord;
-            uint2 positionSS = input.texcoord * _ScreenSize.xy;
+            float2 positionSS = input.texcoord * _ScreenSize.xy;
```

Comment:
This is the gnarliest problem: uint2 was basically destroying any interpolation between samples, since _ScreenSize is much smaller than the viewport of this pixel shader.

Reply:
Indeed shouldn't have been used by the CAS part, but everything else that uses positionSS  should still be uint2 as we use texture loads with it and never samples.

Reply:
Not needed anymore, indeed.

Reply:
This has been removed in new version